src/ludamus/links/db/django/repositories.py

```
@@ -138,6 +153,19 @@
 
 _ISO8601_DURATION_RE = re.compile(r"PT(?:(\d+)H)?(?:(\d+)M)?")
 
+logger = logging.getLogger(__name__)
+
+
+def delete_stored_file(field_file: object, old_name: str) -> None:
+    if (storage := getattr(field_file, "storage", None)) is None:
+        return
+    try:
+        storage.delete(old_name)
+    except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)